checks.d/zk.py

```diff
@@ -129,7 +129,7 @@ def check(self, instance):
         expected_mode = (instance.get('expected_mode') or '').strip()
         tags = instance.get('tags', [])
         cx_args = (host, port, timeout)
-        sc_tags = ["host:{0}".format(host), "port:{0}".format(port)]
+        sc_tags = ["host:{0}".format(host), "port:{0}".format(port)] + tags
```

Comment:
actually a nitpick here: could you `list(set(...))` that please? Just in case...